src/test/cli/rbd/help.t

```diff
@@ -110,6 +113,40 @@
     --io-total arg       total size to write (in B/K/M/G/T)
     --io-pattern arg     write pattern (rand or seq)
   
+  rbd help cg create
+  usage: rbd cg create [--pool <pool>] 
+                       <cg-name> 
```

Comment:
I can't speak for the history, but for now it's based on inertia and staying consistent with the other commands.  If I could start again, I'd vote hyphens between command specs (e.g. cg-create vs cg create) and only named parameters.